collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -1315,7 +1315,7 @@ public void onClick(View v) {
                     FormEntryPrompt[] prompts = formController.getQuestionPrompts();
                     FormEntryCaption[] groups = formController
                             .getGroupsForCurrentIndex();
-                    odkv = new ODKView(this, formController.getQuestionPrompts(),
+                    odkv = new ODKView(this, prompts,
```

Comment:
Please just use one line here. It's too short now.

Reply:
ok. Will do it.